### PR TITLE
updates code for profile workflow

### PR DESCRIPTION
**Pull request by agile-fan430**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202086331093331

### CODE REVIEW

File: src/pages/Home/index.jsx [hunk 1]
```diff
@@ -34,6 +34,11 @@ const Home = (props) => {
             activateWeb3();
         }
     };
+    if (
+        active
+    ) {
+        navigate('/profile/complete-profile');
+    }
```
Comment (NunoMartins21) [inline src/pages/Home/index.jsx:41]: This is not the logic that we want. You're checking whether a wallet is active, so you can trigger the navigation, when you should be checking if the user is or isn't initialized (by checking `userInfo.init`).

Also, this snippet of code should be "encapsulated" inside an `useEffect` hook that gets triggered every time the `userInfo` changes, as we need to know when we're changing users.

But perhaps the most important thing of all is: *why* are we putting this piece of code at `Home`? This is a behavior that should be happening on a larger scale, across all the website. A better place to check whether a user is or isn't `init` is on `src/contexts/UserContext.js`, on the `useEffect` we're using to get user information (lines 307-397).

Let me know if you need more information on what you should do here.

File: src/pages/ProfilePage/components/completeProfile/index.tsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ const CompleteProfile: React.FC = () => {
 		displayName: userInfo?.name || '',
 		userName: userInfo?.username || '',
 		avatar: userInfo?.pfp || '',
-		userBio: userInfo?.bio || '',
+		userBio: userInfo?.bio || 'headline',
```
Comment (NunoMartins21) [inline src/pages/ProfilePage/components/completeProfile/index.tsx:64]: Why? We don't need to show "headline" as default. If the user doesn't insert a headline, no worries. It isn't mandatory.

**Review by NunoMartins21 — CHANGES_REQUESTED** [review]
Hey @agile-fan430, please check my comments here.

**Comment by agile-fan430** [comment]
I have changed all. Pls check

File: src/pages/Home/index.jsx (hunk 1 above)
Comment (agile-fan430) [reply src/pages/Home/index.jsx]: Because it was impossible to implement the Router in UserContext.js file.

File: src/contexts/UserContext.js [hunk 3]
```diff
@@ -372,6 +374,15 @@ export const UserProvider = ({ children }) => {
         JSON.parse(localStorage.getItem('gateway-wallet')) && activateWeb3();
     }, []);
 
+    useEffect(() => {
+        if (userInfo && !userInfo.init) {
+            navigate('/profile/complete-profile');
+        }
+        else if ((userInfo && userInfo.init)) {
+            navigate('/profile');
+        }
```
Comment (NunoMartins21) [inline src/contexts/UserContext.js:383]: We don't want this. I mean, if the user is initialized, then there's no need for them to get redirected to their profiles every time they connect their wallets.

```suggestion
```

File: src/index.js [hunk 4]
```diff
@@ -13,17 +13,18 @@ import { Web3Provider } from '@ethersproject/providers';
...
+import 'react-notifications-component/dist/theme.css';
 
 // AWS
 import Amplify from 'aws-amplify';
 import awsconfig from './aws-exports';
 
 // Analytics
-// import ReactGA from 'react-ga'; // @TODO: Re-add these
+import ReactGA from 'react-ga';
+import { BrowserRouter } from 'react-router-dom';
```
Comment (NunoMartins21) [inline src/index.js:27]: Please remove these lines, as they're not used anymore.

**Review by NunoMartins21 — CHANGES_REQUESTED** [review]
Hey @agile-fan430, it's way better now. However, I've marked you on a few things to change still. Can you take care of them before I push this?

File: src/index.js (hunk 4 above)
Comment (NunoMartins21) [reply src/index.js:27]: Sorry, my mistake. Actually, don't remove the line 27, obviously. Just remove the line 26.

File: src/contexts/UserContext.js (hunk 3 above)
Comment (agile-fan430) [reply src/contexts/UserContext.js:383]: Sanket told me that the page should redirect to profile page for the already registered users.
So I considered that case.
If you need to remove it, no problem. :)
I will remove and commit again.

**Comment by agile-fan430** [comment]
Nuno. I have changed and pushed. pls check.

File: src/contexts/UserContext.js (hunk 3 above)
Comment (NunoMartins21) [reply src/contexts/UserContext.js]: Hey, I was checking this with Sanket. Let's not have this. I noticed you removed it, so I'm just going to accept.

**Review by NunoMartins21 — APPROVED** [review]
Sorry, I took some time to pick this up. In any case, this looks really good and works based on my tests. Nicely done. Approved and merged.

**Pull request merged**